parsl/monitoring/monitoring.py

```diff
@@ -83,6 +84,9 @@ def __init__(self,
              If set to 0, only start and end information will be logged, and no periodic monitoring will
              be made.
              Default: 30 seconds
+        capture_file_provenance : bool
```

Comment:
Name this either `file_provenance` (my preference - no need for a verb). If you really want to use a verb, use `enabled` at the end, for consistency with `resource_monitoring_enabled`, but I personally prefer no verb.

Reply:
ok, fixed